components/brave_new_tab_ui/components/search/SearchDialog.tsx

```diff
@@ -64,7 +63,7 @@ const Dialog = styled.dialog<{ offsetY: number }>`
 
   outline: none;
   border: none;
-  border-radius: ${searchBoxRadius};
```

Comment:
Hey, I think this should be reverted too - if these two things don't share a radius the UI will look super weird (with the background blur effect).

Sharing a variable means that if we go and tweak this later it'll be pretty obvious that the two things are tied together.